packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -255,15 +261,42 @@ class AppBar extends StatefulWidget implements PreferredSizeWidget {
 
   /// The primary widget displayed in the app bar.
   ///
-  /// Typically a [Text] widget containing a description of the current contents
-  /// of the app.
+  /// Typically a [Text] widget that contains a description of the current
+  /// contents of the app.
+  ///
+  /// Becomes the middle component of the [NavigationToolBar] built by this widget.
+  /// The [title]'s width is constrained to fit within the toolbar's [leading]
```

Comment:
I think this is the correct description, unless I'm misunderstanding
```suggestion
  /// The [title]'s width is constrained to fit in the remaining space between the toolbar's [leading]
```

Reply:
Good point, updated.